### PR TITLE
Fixing some issues with multi-cluster and unifiying some stuff across the recipes

### PR DESCRIPTION
**Pull request by boredabdel**
- Unifying bash across all examples
- Fixing some typos
- Making the whereami spit out headers for the asm-https-e2e recipes

### CODE REVIEW

File: ingress/multi-cluster/mci-https-e2e/ingress.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ metadata:
   name: foobar-ingress
   namespace: multi-cluster-demo
   annotations:
-    networking.gke.io/static-ip: x.x.x.x
+    networking.gke.io/static-ip: 34.149.198.76
```
Comment (mikouaj) [inline ingress/multi-cluster/mci-https-e2e/ingress.yaml:7]: Do you want to keep hardcoded IP here ?

**Review by mikouaj — APPROVED** [review]
LGTM! Please check my minor comments.

File: ingress/multi-cluster/mci-basic/README.md [hunk 2]
```diff
@@ -109,33 +109,31 @@ Now that you have the background knowledge and understanding of MCI, you can try
...
-
-    There are two manifests in this folder:
+3. Deploy the two clusters `gke-1` and `gke-2` as specified in [cluster setup](../../../cluster-setup.md#Multi-cluster-environment-basic) and follow the steps for cluster registration with Hub and enablement of Multi-cluster Ingress.
 
+There are two manifests in this folder:
     - app.yaml is the manifest for the foo and bar Deployments. This manifest should be deployed on both clusters.
     - ingress.yaml is the manifest for the MultiClusterIngress and MultiClusterService resources. These will be deployed only on the `gke-1` cluster as this was set as the config cluster and is the  cluster that the MCI controlller is listening to for updates.
 
-5. Separately log in to each cluster and deploy the app.yaml manifest. You can configure these contexts as shown [here](../../../cluster-setup.md).
+1. Separately log in to each cluster and deploy the app.yaml manifest. You can configure these contexts as shown [here](../../../cluster-setup.md).
```
[the remark this answers is not in the document]
Comment (boredabdel) [reply ingress/multi-cluster/mci-basic/README.md:134]: Fixed/

File: ingress/multi-cluster/mci-https-e2e/ingress.yaml (hunk 1 above)
Comment (boredabdel) [reply ingress/multi-cluster/mci-https-e2e/ingress.yaml:7]: Not really. Fixed ;)

File: ingress/multi-cluster/mci-https-e2e/README.md [hunk 3]
```diff
@@ -28,12 +28,14 @@
 
 This recipe demonstrates deploying Multi-cluster Ingress across two clusters to expose two different Services hosted across both clusters. The cluster `gke-1` is in `REGION#1` and `gke-2` is hosted in `REGION#2`, demonstrating multi-regional load balancing across clusters. All Services will share the same MultiClusterIngress and load balancer IP, but the load balancer will match traffic and send it to the right region, cluster, and Service depending on the request.
 
+[//]: # (TODO - abdelfettah@: add diagram)
```
[the remark this answers is not in the document]
Comment (boredabdel) [reply ingress/multi-cluster/mci-https-e2e/README.md:31]: That a good point. Done ;)

**Pull request merged**